pyvista/common.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, *args, **kwargs):
         self._point_bool_array_names = []
         self._cell_bool_array_names = []
         self._field_bool_array_names = []
+        self._repeat_texture = False
```

Comment:
Let's avoid adding another property to meshes since the underlying VTK object has no way of keeping track of this

Reply:
Also, I often deal with more than one texture per mesh so I'm thinking there will definitely be cases where users want to repeat one texture but not others when plotting the same mesh in different routines